csm/core/agent/api.py

```diff
@@ -151,7 +151,7 @@ def error_response(err: Exception, request) -> dict:
             resp["message"] = f'{str(err)}'
 
         audit = CsmRestApi.http_request_to_log_string(request)
-        if hasattr(request,"session"):
+        if hasattr(request,"session") and request.session is not None:
```

Comment:
May be you need complete check here -
if ((hasattr(request, "session") and request.session is not None) and
            (hasattr(request.session, "credentials") and request.session.credentials is not None)):

I saw this in one of the pending PRs by Tadeu.

Reply:
Np, done.